scripts/postbuild.sh

```diff
@@ -5,11 +5,12 @@ cp third_party/zxing-js.umd.min.js minified/html5-qrcode.min.js
 echo 'Copied third_party/zxing-js.umd.min.js to minified/html5-qrcode.min.js'
 
 ## Copy all other temp files to final minified script
+
+printf '\n\n/** Html5Qrcode **/\n' >> minified/html5-qrcode.min.js
 cat minified/html5-qrcode.tmp.js >> minified/html5-qrcode.min.js
 echo 'Copied minified/html5-qrcode.tmp.js to minified/html5-qrcode.min.js'
```

Comment:
It'd be nicer to consistently use `printf` across the script for consistency and to avoid confusion on when to use what.

Reply:
Well actually, `printf` comes with the benefit for having more control of the formatting, at the cost of having to do more formatting. For normal printing to the commandline using `echo` is much more straightforward as it adds a trailing new line. Otherwise
```bash
echo 'Initiating post-build sequence'
```
would need to be turned into
```bash
printf 'Initiating post-build sequence\n'
```
which I feel only adds complexity and error proniness. Are you sure you want me to change all `echo`´s?